### PR TITLE
[Refactor] Remove New Relic references

### PR DESCRIPTION
**Pull request by damassi**
Removes 
```
NEW_RELIC_APP_NAME=force-production
NEW_RELIC_LOG_LEVEL=error
NEW_RELIC_CAPTURE_PARAMS=true
NEW_RELIC_LICENSE_KEY=119b480b649a5b9919c6accba32ba47992262908
NEW_RELIC_BROWSER_MONITOR_ENABLE=true
```

from env vars and some left-behind code.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -1,336 +0,0 @@
-{
```
Comment (damassi) [inline app.json]: This is old Heroku stuff

**Comment by joeyAghion** [comment]
👍 from me.

It looks like you already unset these in production (right?). Staging can also have its `NEW_RELIC_LOG` var unset.

**Comment by joeyAghion** [comment]
Actually, a follow-up question: where is `newRelicHead` defined?

**Comment by damassi** [comment]
`newRelicHead` was just a variable, so if an app needed to include something like
```
newRelicHead = 'some value' 
```
would mix it in.

**Pull request merged**